pkg/reconciler/delivery/table_test.go

```diff
@@ -91,6 +91,21 @@ func Configuration(namespace, name string, co ...ConfigOption) *v1.Configuration
 	return c
 }
 
+// Revision creates a revision with RevisionOptions
+func Revision(namespace, name string, ro ...RevisionOption) *v1.Revision {
```

Comment:
nit: these could be moved to their own test helper.

Reply:
I put this in `table_test.go` because I expect it to be used by table tests later. By "their own test helper" do you mean a separate file e.g. `test_helpers.go`?

Reply:
yeah something like `resources.go` in https://github.com/googleinterns/knative-continuous-delivery/tree/main/pkg/reconciler/testing